datasets/modeled/transport/dimension_mappings/model_year_to_model_year.csv

```diff
@@ -0,0 +1,11 @@
+from_id,to_id,fraction
+2025,2024,0.7686800739128512
+2025,2026,0.6063048435273051
+2030,2026,0.17990135573363503
+2025,2028,1.1175065070414483
+2030,2028,0.33158391822885314
```

Comment:
agreed it's unintuitive because you might expect summation to 1 or more emphasis on 2030 but the growth is non-linear so the numbers are funny indeed.
In fact, I believe there are infinite combinations of the 2025 and 2030 load that would add up to the 2028 total load. I simply picked 50% of each of 2025 and 2030 that would make up 100% of 2028. I believe this will make the 2026 and 2028 load heterogeneity (the distribution between different subsectors, metrics, geography, and time, etc.) will look like both 2025 and 2030, rather than look like only of one or the other (e.g. if I only scaled 2025 load to get at 2026 load).

Not sure if this should be formalized somewhere i.e. some notes on how to determine appropriate fractions for interpolating data between years, especially when growth is non-linear.

Reply:
now I'm wondering if I should set the fractions for 2026 to be the sum of 80% of 2025 scaled for 2026 + 20% of 2030 scaled for 2026 
instead of 50% of 2025 scaled for 2026 + 50% of 2030 scaled for 2026 
(either of above should be better than simply 2025 scaled for 2026, or simply 2030 scaled for 2026)

Reply:
So for 2026:
- Fix the 2025 fraction to 0.8
- Start the 2030 fraction to 0.2 and then adjust to match the annual energy from Task 2
?

And for 2028:
- Fix the 2025 fraction to 0.4
- Start the 2030 fraction to 0.6 and then adjust to match the annual energy from Task 2
?

If that's what you mean and it's easy to implement: yes, I like that better.

Reply:
not quite. I have now adjusted 2026 to 80% 2025 scaled to 2026 + 20% 2030 scaled to 2026, and that ends up as 97% 2025 + 7% 2030, which is a better representation of how loads are distributed between subsectors/metrics/types etc. (closer to 2025).